app/src/MainController.cpp

```diff
@@ -113,18 +138,17 @@ void MainController::draw_moon() {
     shader->set_mat4("projection", graphics->projection_matrix());
     shader->set_mat4("view", graphics->camera()->view_matrix());
 
-
-    float timeValue        = platform->frame_time().current;
-    //float pom = (cos(timeValue) + 1)/2.f; // fade function
+    float timeValue = platform->frame_time().current;
+    // float pom = (cos(timeValue) + 1)/2.f; // fade function
     if (moon_event == 1) {
-        this->light.intensity = glm::vec3(0.2);
+        this->m_light.intensity = glm::vec3(0.2);
```

Comment:
`this->` se u C++ ne piše, uvek je implicitno, osim ako su argument i polje istog naziva, mada se i to izbegava. 
Možete slobodno obrisati `this->` iz celog fajla.